berkdb/txn/txn_auto.c

```diff
@@ -78,6 +80,9 @@ __txn_regop_log_int(dbenv, txnid, ret_lsnp, ret_contextp, flags,
 #endif
 
 	rectype = DB___txn_regop;
```

Comment:
gbl_utxnid_log is toggle-able at runtime- copy the tunable at top & used the copied version

Reply:
[Done here](https://github.com/bloomberg/comdb2/pull/3712/commits/2669ea1c2e1d2464f893eb2bfd8c4053b8538066).